WordPress/src/main/java/org/wordpress/android/ui/posts/PostListUploadStatusTracker.kt

```diff
@@ -4,10 +4,6 @@ import org.wordpress.android.fluxc.model.PostModel
 import org.wordpress.android.fluxc.model.SiteModel
 import org.wordpress.android.fluxc.store.UploadStore
 import org.wordpress.android.ui.uploads.UploadActionUseCase
-import org.wordpress.android.ui.uploads.UploadActionUseCase.UploadAction.DO_NOTHING
-import org.wordpress.android.ui.uploads.UploadActionUseCase.UploadAction.REMOTE_AUTO_SAVE
-import org.wordpress.android.ui.uploads.UploadActionUseCase.UploadAction.UPLOAD
-import org.wordpress.android.ui.uploads.UploadActionUseCase.UploadAction.UPLOAD_AS_DRAFT
 import org.wordpress.android.ui.uploads.UploadService
 import org.wordpress.android.viewmodel.posts.PostListItemUploadStatus
```

Comment:
I agree the name `Tracker` is a bit misleading/confusing. Especially since we use `Tracker` suffix for feature specific Tracks/Analytics classes.
I feel like caching statuses in a "Builder" would be confusing as well. Perhaps something like "Provider/Manager/.." -> or create two separate classes `PostListUploadStatusBuilder` and `PostListUploadStatusCache`

>Should we remove that “This is a temporary class” comment?

:D it's still a "temporary" class -> the plan was to refactor it when we have more time, but we never did. I guess we can remove it ;).

Reply:
If it's still temporary, we can hold off renaming it and making more changes. 🙂